src/org/zaproxy/zap/extension/advreport/resources/help/contents/advreport.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+<META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=utf-8">
+<TITLE>
+Customizable HTML Report
+</TITLE>
+</HEAD>
+<BODY BGCOLOR="#ffffff">
+
+<H1>Customizable HTML Report</H1>
+The advrepot extension allows you to control the information that goes into the ZAP scanning report.
```

Comment:
There's a typo "advrepot".
I'd suggest changing to "The Customizable HTML Report add-on allows..." (change the name of the add-on accordingly, it's more descriptive than "AdvReport", and change the index entry).